pkg/cmd/server/admission/init.go

```diff
@@ -38,3 +40,19 @@ func Validate(plugins []admission.Interface) error {
 	}
 	return nil
 }
+
+// GetPluginConfigFile translates from the master plugin config to a file name containing
+// a particular plugin's config (the file may be a temp file if config is embedded)
+func GetPluginConfigFile(pluginConfig map[string]api.AdmissionPluginConfig,
+	pluginName string, defConfigFile string) (string, error) {
```

Comment:
perhaps there should be a coding style document specifying at what point a line break is indicated? I was just keeping that one line from sticking out from the rest of the file...